packages/aws-cdk/lib/init-templates/v1/app/typescript/cdk.template.json

```diff
@@ -1,3 +1,18 @@
 {
-  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts"
+  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "jest.config.js",
+      "tsconfig.json",
+      "yarn.lock",
+      "package*.json",
```

Comment:
This order seems inconsistent with the JS templates. I think the JS template order is the correct one we want to use here (first `"package*.json"`, then `"yarn.lock"`).